sdk/arm/keyvault/2019-09-01/armkeyvault/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+Generated from https://github.com/Azure/azure-rest-api-specs/tree/9563d064dabcf535e35a52bc2e0e3ada4e74e64f
+
+Code generator @autorest/go@4.0.0-preview.3
```

Comment:
Does this need to be regenerated with preview.5?  Or maybe that happened and the CHANGELOG wasn't updated?

Reply:
Done. Used preview.6.